### ISSUE TITLE
How to resume SAM in pytorch?

### ISSUE THREAD

**Issue by whynique** [davda54/sam#34]
Hi!
Thanks for your great work! When I follow your instructions and insert SAM in my code, it works well. But the training loss collapses every time I resume the model. So I suspect that I missed some critical steps to resume SAM. Do you have some suggestions?

**Comment by TimDarcet** [comment]
I had the same issue and fixed it by redefining load_state_dict in the SAM class. This PR should fix it.
